src/components/[guild]/CreateFormModal/components/Display/Choice.tsx

```diff
@@ -0,0 +1,101 @@
+import {
+  Checkbox,
+  CheckboxGroup,
+  CheckboxGroupProps,
+  CheckboxProps,
+  HStack,
+  Input,
+  Radio,
+  RadioGroup,
+  RadioGroupProps,
+  RadioProps,
+  Stack,
+  forwardRef,
+} from "@chakra-ui/react"
+import { ComponentType, useState } from "react"
+import { useFormContext } from "react-hook-form"
+import { CreateFieldParams } from "../../schemas"
+
+type Props = {
+  field: CreateFieldParams
+} & (RadioGroupProps | CheckboxGroupProps)
+
+const GroupComponents: Record<
+  "SINGLE_CHOICE" | "MULTIPLE_CHOICE",
+  ComponentType<RadioGroupProps | CheckboxGroupProps>
+> = {
+  SINGLE_CHOICE: RadioGroup,
+  MULTIPLE_CHOICE: CheckboxGroup,
+}
+
+const FieldComponents: Record<
+  "SINGLE_CHOICE" | "MULTIPLE_CHOICE",
+  ComponentType<RadioProps | CheckboxProps>
+> = {
+  SINGLE_CHOICE: Radio,
+  MULTIPLE_CHOICE: Checkbox,
+}
+
+// TODO: should we pass that ref down? (probably for focus management?)
```

Comment:
Unfortunately `RadioGroup` & `CheckboxGroup` don't accept a ref, so I didn't want to spend more time on this, but please lmk if you can think about a possible solution for this